docs(kb): why comark (#138)
comarkdown/comark: docs/content/1.getting-started/0.introduction.md

::
```

The `::alert` is a block component that supports properties and children (also known as slots).

::tip{to="/kb/why-comark"}

> Want to know more about the Comark design and why we've created it? [Why Comark](/kb/why-comark) explains the reasoning behind this design choice.

Reviewer: ```suggestion
Learn why we created Comark and the principles behind its design in [Why Comark](https://github.com/kb/why-comark).
```
